src/Algebra/Graph.hs

```diff
@@ -268,13 +279,15 @@ apR :: Graph (a -> b) -> Graph a -> Graph b
 apR f x = bindR f (<$> x)
 {-# INLINE apR #-}
 
+-- | `>>=` is a good consumer and producer.
 instance Monad Graph where
     return = pure
     (>>=)  = bindR
 
 bindR :: Graph a -> (a -> Graph b) -> Graph b
-bindR g f = foldg Empty f Overlay Connect g
-{-# INLINE [0] bindR #-}
+bindR g f = buildg $ \e v o c ->
```

Comment:
Do we still need `bindR`? I couldn't find any remaining rules for it.

Reply:
Indeed we can remove them, the same apply to `apR` and `fmapR`, they were leftovers.